Source/WebCore/rendering/LegacyInlineBox.cpp

```diff
@@ -190,6 +192,23 @@ void LegacyInlineBox::adjustPosition(float dx, float dy)
         downcast<RenderBox>(renderer()).move(LayoutUnit(dx), LayoutUnit(dy));
 }
 
+void LegacyInlineBox::adjustBlockDirectionPosition(float delta, bool adjustStaticPosition)
+{
+    if (isHorizontal())
+        LegacyInlineBox::adjustPosition(0, delta);
+    else
+        LegacyInlineBox::adjustPosition(delta, 0);
+
+    if (adjustStaticPosition && renderer().isOutOfFlowPositioned()) {
+        CheckedRef<RenderBox> box = downcast<RenderBox>(renderer());
```

Comment:
```suggestion
        CheckedRef box = downcast<RenderBox>(renderer());
```

The type is already in the downcast.